source/api/presentation/3.0/index.md

```diff
@@ -123,26 +123,25 @@ Content resources such as images, audio, video or text that are associated with
 
 An ordered list of Manifests, and/or further Collections.  Collections allow easy navigation among the Manifests in a hierarchical structure, potentially each with its own descriptive information.
 
-##### AnnotationPage
+##### Annotation Page
 {: #overview-annotationpage}
 
-An ordered list of Annotations in a single response, typically associated with a single Canvas, and can be part of an AnnotationCollection.
+An ordered list of Annotations in a single response, typically associated with a single Canvas, and can be part of an Annotation Collection.
 
 ##### Annotation
 {: #overview-annotation}
 
-Content resources and commentary are associated with a Canvas via an Annotation.  This provides a single, coherent method for aligning information, and provides a standards based framework for distinguishing parts of resources and parts of Canvases.  As Annotations can be added later, it promotes a distributed system in which publishers can align their content with the descriptions created by others.
+Content resources, commentary and other references are associated with a Canvas via Annotations.  This provides a single, coherent method for aligning information, and provides a standards based framework for distinguishing parts of resources and parts of Canvases.  As Annotations can be added later, it promotes a distributed system in which publishers can align their content with the descriptions created by others.
```

Comment:
Start this sentence with "All"?

Reply:
Or something like... 

All content is linked to the Canvas via Annotations. The same mechanism is used for the visible or audible resources as is used for transcriptions, commentary, tags and other content. This provides a single, coherent...

I know this isn't adding anything new. Just reinforcing the point.

Reply:
:+1: